101-aks/metadata.json

```diff
@@ -4,7 +4,7 @@
     "itemDisplayName": "Azure Container Service (AKS)",
     "description": "Deploy a managed cluster with Azure Container Service (AKS)",
     "summary": "Deploy a managed cluster with Azure Container Service (AKS)",
-    "githubUsername": "vyta",
-    "dateUpdated": "2019-03-21"
+    "githubUsername": "montebhoover",

```

Comment:
do you want to take ownership of this sample?

Reply:
Ah I didn't realize that's what I was signifying there.  I don't mind taking ownership, but I can undo that line if you prefer.